src/app_errors/mod.rs

```diff
@@ -2,16 +2,13 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, you can obtain one at https://mozilla.org/MPL/2.0/.
 
-use std::{collections::HashMap, result};
+use std::collections::HashMap;
 
-use failure::Error;
 use rocket_contrib::Json;
 
 #[cfg(test)]
 mod test;
 
-pub type AppResult<T> = result::Result<T, Error>;
```

Comment:
Is the plan to use `AppResult` more widely than just the logging module eventually? Might be worth keeping here if that's the case. (I confess I don't really get it what it's for at the moment)

Reply:
Yeah, so, that is needed for the logging and I think the plan is not to use it anywhere else. 

Since this code came from megaphone there are some decisions that they made that don't make sense to us, I guess. Probably they use this type in their error functions, but we just use it in the logging. That's why I moved it.

(I also don't get it 100%, I could investigate and try to understand it).

Reply:
AppResult isn't needed here currently since MozlogLogger::new here never returns an Error

megaphone's method can error out in a couple cases, one of them being determining the current hostname. You may also want to lookup the hostname similarly (MozLog standard recommends including one: https://wiki.mozilla.org/Firefox/Services/Logging )

Reply:
I'll remove this for now and just use the normal `Result` type then. I'll take a closer look at error handling in #48 :)